resources/client_install.rb

```diff
@@ -22,7 +22,7 @@
 action :install do
   mariadb_repository 'Add mariadb.org repository' do
     version new_resource.version
-    only_if { new_resource.setup_repo }
+    only_if { !new_resource.setup_repo.nil? && new_resource.setup_repo }
```

Comment:
By saying `new_resource.setup_repo` you are implying it is not `nil` so you don't need to say `new_resource.setup_repo`; this change can be reverted.

Reply:
indeed, it's already true or false, never nil.